FogCreek/Glitch-Community
# Cassey-community: skip section buttons in project lists

## Links
* cassey-community.glitch.me
* https://github.com/FogCreek/Glitch-Community-Work/issues/153
* Only on project list sections, not collection sections yet

## GIF/Screenshots:
### When you tab to just before the place where any projects-list component renders projects: 
![image](https://user-images.githubusercontent.com/4480480/61739326-d3ddde80-ad51-11e9-9707-4ed3f3d7495a.png)

###  When you aren't in the exactly correct tab location, the button is hidden visually. 
![image](https://user-images.githubusercontent.com/4480480/61739482-1dc6c480-ad52-11e9-9e6d-86c92fc2e604.png)

### It appears on the collection page projects list, too
![image](https://user-images.githubusercontent.com/4480480/61739503-2919f000-ad52-11e9-9f60-218e9d5ee3ba.png)

### And on every section of projects (but not deleted projects) on the profile page
![image](https://user-images.githubusercontent.com/4480480/61739543-3f27b080-ad52-11e9-9b0a-6b5904fbaa84.png)

### The same button appears after the projects and lets you jump to before the projects
![image](https://user-images.githubusercontent.com/4480480/61739554-464ebe80-ad52-11e9-9ebc-9379f325a71b.png)


## Changes:
* Creates a component that wraps another component and adds buttons that let you move your keyboard focus to the other button in the pair when you click the button. Each button only appears when you are focusing on it. See the issue in Glitch-Community-Work for details. 

## How To Test:
* Tab down the homepage while logged in, you should see the buttons in your recent projects section and they should work
* tab through a collection page
* tab through a user profile page
* try out the buttons. You should have a focus jump with no page refresh

## Feedback I'm looking for:
* Visually, could these be placed better when the button appears?
